docs/_user_guide/best_practices_migrating_from_jupyter.md

```diff
@@ -79,37 +85,37 @@ In JupyterHub, you should see the new kernel, `my_env`, available.
 
 Try running your JupyterHub notebook using the `my_env` kernel you just installed.  You can change the kernel of an existing notebook by going to Kernel->Change Kernel...
 
-<img src="/images/jupyterhub_changekernel.png" alt="Change Kernel">
+![Change Kernel](/assets/images/jupyterhub_changekernel.png)
 
 If it works on this kernel, your next step would be to migrate these commands to a `.py` script.  You can test this by activating your `venv` environment on the Yens, and running your script via `python <my_script.py>`.
 
 #### Write a submit script to run your process on `yen-slurm` using your working environment
```

Comment:
Can we make the title shorter? I would suggest making all titles a few words long so they fit nicely in the table of contents on the side

Reply:
done